\hypertarget{classfc_array_index_out_of_bound_exception}{}\doxysection{fc\+Array\+Index\+Out\+Of\+Bound\+Exception类 参考}
\label{classfc_array_index_out_of_bound_exception}\index{fcArrayIndexOutOfBoundException@{fcArrayIndexOutOfBoundException}}


{\ttfamily \#include $<$fcexceptions.\+hpp$>$}



类 fc\+Array\+Index\+Out\+Of\+Bound\+Exception 继承关系图\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=270pt]{d7/dbf/classfc_array_index_out_of_bound_exception__inherit__graph}
\end{center}
\end{figure}


fc\+Array\+Index\+Out\+Of\+Bound\+Exception 的协作图\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=270pt]{dc/d39/classfc_array_index_out_of_bound_exception__coll__graph}
\end{center}
\end{figure}
\doxysubsection*{Public 成员函数}
\begin{DoxyCompactItemize}
\item 
\mbox{\hyperlink{classfc_array_index_out_of_bound_exception_a53ae0a4c8b58386f5afe970d533b1ebc}{fc\+Array\+Index\+Out\+Of\+Bound\+Exception}} (const char $\ast$why\+Exception=\char`\"{}the array index is out of bound.\char`\"{})
\end{DoxyCompactItemize}
\doxysubsubsection*{Public 成员函数 继承自 \mbox{\hyperlink{classfc_runtime_exception}{fc\+Runtime\+Exception}}}
\begin{DoxyCompactItemize}
\item 
\mbox{\hyperlink{classfc_runtime_exception_a264ae184e45fbc84cc58952ba062c5f5}{fc\+Runtime\+Exception}} (const char $\ast$why\+Exception=\char`\"{}runtime exception occured\char`\"{})
\item 
const char $\ast$ \mbox{\hyperlink{classfc_runtime_exception_ac9faee48241584efdec3f0e2955623b1}{what}} ()
\end{DoxyCompactItemize}


\doxysubsection{详细描述}


在文件 \mbox{\hyperlink{fcexceptions_8hpp_source}{fcexceptions.\+hpp}} 第 \mbox{\hyperlink{fcexceptions_8hpp_source_l00052}{52}} 行定义.



\doxysubsection{构造及析构函数说明}
\mbox{\Hypertarget{classfc_array_index_out_of_bound_exception_a53ae0a4c8b58386f5afe970d533b1ebc}\label{classfc_array_index_out_of_bound_exception_a53ae0a4c8b58386f5afe970d533b1ebc}} 
\index{fcArrayIndexOutOfBoundException@{fcArrayIndexOutOfBoundException}!fcArrayIndexOutOfBoundException@{fcArrayIndexOutOfBoundException}}
\index{fcArrayIndexOutOfBoundException@{fcArrayIndexOutOfBoundException}!fcArrayIndexOutOfBoundException@{fcArrayIndexOutOfBoundException}}
\doxysubsubsection{\texorpdfstring{fcArrayIndexOutOfBoundException()}{fcArrayIndexOutOfBoundException()}}
{\footnotesize\ttfamily fc\+Array\+Index\+Out\+Of\+Bound\+Exception\+::fc\+Array\+Index\+Out\+Of\+Bound\+Exception (\begin{DoxyParamCaption}\item[{const char $\ast$}]{why\+Exception = {\ttfamily \char`\"{}the~array~index~is~out~of~bound.\char`\"{}} }\end{DoxyParamCaption})\hspace{0.3cm}{\ttfamily [inline]}}



在文件 \mbox{\hyperlink{fcexceptions_8hpp_source}{fcexceptions.\+hpp}} 第 \mbox{\hyperlink{fcexceptions_8hpp_source_l00055}{55}} 行定义.


\begin{DoxyCode}{0}
\DoxyCodeLine{00055\ :\ \mbox{\hyperlink{classfc_runtime_exception}{fcRuntimeException}}(whyException)\ \{\};}

\end{DoxyCode}
